packages/getting-started/src/browser/getting-started-preferences.ts

```diff
@@ -22,21 +22,32 @@ import {
     PreferenceSchema,
     PreferenceContribution
 } from '@theia/core/lib/browser/preferences';
+import { FrontendApplicationConfigProvider } from '@theia/core/lib/browser/frontend-application-config-provider';
 import { nls } from '@theia/core/lib/common/nls';
 
 export const GettingStartedPreferenceSchema: PreferenceSchema = {
     'type': 'object',
     properties: {
-        'welcome.alwaysShowWelcomePage': {
```

Comment:
Do you mind adding a breaking  changes note in our [changelog](https://github.com/eclipse-theia/theia/blob/master/CHANGELOG.md#v1410--) for v1.41.0 about the preference being removed in favor of `workbench.startupEditor`?